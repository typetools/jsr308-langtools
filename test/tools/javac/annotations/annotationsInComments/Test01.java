/*
 * @test
 * @bug 1234567
 * @summary Annotations in comments are recognized
 * @author Werner Dietl
 * @compile/fail/ref=Test01.out -XDrawDiagnostics Test01.java
 */
class Test01 {
    // Recognized comment
    /*@AnnoString("Hi There!")*/
    void t01() {}
    // Recognized comment, bad symbol
    /*@AnnoStringBad("Hi There!")*/
    void t02() {}

    // Recognized comment
    /*@AnnoString("HiThere(((")*/
    void t03() {}
    // Recognized comment, bad symbol
    /*@AnnoStringBad("HiThere(((")*/
    void t04() {}
    // Recognized comment, bad type
    /*@AnnoInt("HiThere(((")*/
    void t05() {}

    // Recognized comment, bad type
    /*@AnnoStringBad("HiThere) ")*/
    void t06() {}

    @interface AnnoString {
	String value();
    }

    @interface AnnoInt {
	int value();
    }
}
