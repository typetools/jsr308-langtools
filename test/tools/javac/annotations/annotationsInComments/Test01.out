Test01.java:13:8: compiler.err.cant.resolve.location: kindname.class, AnnoStringBad, , , (compiler.misc.location: kindname.class, Test01, null)
Test01.java:20:8: compiler.err.cant.resolve.location: kindname.class, AnnoStringBad, , , (compiler.misc.location: kindname.class, Test01, null)
Test01.java:23:16: compiler.err.prob.found.req: (compiler.misc.inconvertible.types: java.lang.String, int)
Test01.java:27:8: compiler.err.cant.resolve.location: kindname.class, AnnoStringBad, , , (compiler.misc.location: kindname.class, Test01, null)
4 errors